dao/src/main/java/org/thingsboard/server/dao/sql/query/DefaultQueryLogComponent.java

```diff
@@ -33,8 +39,72 @@ public class DefaultQueryLogComponent implements QueryLogComponent {
     @Override
     public void logQuery(QueryContext ctx, String query, long duration) {
         if (logSqlQueries && duration > logQueriesThreshold) {
-            log.info("QUERY: {} took {} ms", query, duration);
-            Arrays.asList(ctx.getParameterNames()).forEach(param -> log.info("QUERY PARAM: {} -> {}", param, ctx.getValue(param)));
+
+            String sqlToUse = substituteParametersInSqlString(query, ctx);
+            log.warn("SLOW QUERY took {} ms: {}", duration, sqlToUse);
+
```

Comment:
Usually, we don't put extra spaces before and after code blocks. This is a minor comment FYI and it is not blocking the merge of this PR.